opencensus/trace/ext/grpc/client_interceptor.py

```diff
@@ -50,41 +50,39 @@ class OpenCensusClientInterceptor(grpc.UnaryUnaryClientInterceptor,
                                   grpc.StreamUnaryClientInterceptor,
                                   grpc.StreamStreamClientInterceptor):
 
-    def __init__(self, tracer=None, host_port=None):
```

Comment:
Please keep the `tracer` parameter here since we also need to handle the case that sharing the tracer between threads. See #133, which was addressing issue #134 caused by `tracer` not shared between threads when instrumenting google cloud client library calls.

Reply:
Will do! 

Would the current integration in `google_cloud_clientlibs` be facing this same issue if it is passing in the tracer at initialization time? For instance, if a Flask endpoint calls a gcloud service, would the gcloud span be nested under the flask one?